conda_build/utils.py

```diff
@@ -2139,3 +2237,6 @@ def is_conda_pkg(pkg_path: str) -> bool:
 
 def package_record_to_requirement(prec: PackageRecord) -> str:
     return f"{prec.name} {prec.version} {prec.build}"
+
+
+log = get_logger(__name__)
```

Comment:
Why is this at the bottom?

Reply:
Because the function `get_logger` is only defined mid-module, but I think we don't need this `log` object after all. Every function is initializing their own logger anyway (yes, I think that has a non-negligible cost, but that's a subject for other PR). I'll remove it.